modules/ipi-install-configuring-networking.adoc

```diff
@@ -6,109 +6,166 @@
 [id="configuring-networking_{context}"]
 = Configuring networking
 
-Before installation, you must configure the networking on the provisioner node. Installer-provisioned clusters deploy with a bare-metal bridge and network, and an optional provisioning bridge and network.
+Before installation, you must configure networking settings for the provisioner node. Installer-provisioned clusters deploy with a bare-metal bridge and network resources, and an optional provisioning bridge and network resources.
 
 image::210_OpenShift_Baremetal_IPI_Deployment_updates_0122_1.png[Configure networking]
 
 [NOTE]
 ====
-You can also configure networking from the web console.
+You can also configure networking settings from the {product-title} web console.
 ====
 
-.Procedure
+.Prerequisites
 
-. Export the bare-metal network NIC name by running the following command:
-+
-[source,terminal]
-----
-$ export PUB_CONN=<baremetal_nic_name>
-----
+* You installed the `nmstate` package with the `sudo dnf install -y <package_name>` command. The package includes the `nmstatectl` CLI.
+
+.Procedure
 
 . Configure the bare-metal network:
 +
 [NOTE]
 ====
-The SSH connection might disconnect after executing these steps.
+When configuring the bare-metal network and the secure shell (SSH) connection disconnects, NMState has a rollback mechanism that automatically reverts any configurations. You can also use the `nmstatectl gc` tool to generate configuration files for specified network state files.
 ====
-
-.. For a network using DHCP, run the following command:
 +
-[source,terminal]
+.. For a network that uses Dynamic Host Configuration Protocol (DHCP), create an NMState YAML file and specify the bare-metal bridge interface and any physical interfaces in the file:
++
+.Example bare-metal bridge interface configuration that uses DHCP
+[source,yaml]
+----
+# ...
+interfaces:
+  - name: eth0
+    type: ethernet
+    state: up
+    ipv4:
+      enabled: false
+    ipv6:
+      enabled: false
+  - name: baremetal
+    type: linux-bridge
+    state: up
+    bridge:
+      options:
+        stp:
+          enabled: false
+      port:
+        - name: eth0
+# ...
 ----
-$ sudo nohup bash -c "
-    nmcli con down \"$PUB_CONN\"
-    nmcli con delete \"$PUB_CONN\"
-    # RHEL 8.1 appends the word \"System\" in front of the connection, delete in case it exists
-    nmcli con down \"System $PUB_CONN\"
-    nmcli con delete \"System $PUB_CONN\"
-    nmcli connection add ifname baremetal type bridge <con_name> baremetal bridge.stp no <1>
```

Comment:
So NMstate will not delete legacy `/etc/sysconfig/network-scripts/ifcfg-eth0`  `System eth0` style profiles.  So we may still need these lines.

```
    nmcli con down \"System $PUB_CONN\"
    nmcli con delete \"System $PUB_CONN\"
```

The problem only occurs on reboot.  The old legcay profiles can activate first and then we can't attach `eth0` to the baremetal bridge.

I need to do more testing, so put this on hold for now.

Sorry for the delay.